planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/data_manager.cpp

```diff
@@ -94,23 +90,13 @@ bool DataManager::getLaneChangeApproval()
 bool DataManager::getForceLaneChangeSignal()
 {
   constexpr double timeout = 0.5;
-  if (clock_->now() - force_lane_change_.stamp > rclcpp::Duration::from_seconds(timeout)) {
+  if (clock_->now() - force_lane_change_.stamp > rclcpp::Duration(timeout)) {
```

Comment:
I guess it's better to use from_seconds.

Reply:
Fixed.